src/qibolab/result.py

```diff
@@ -38,7 +38,12 @@ def magnitude(self):
     @cached_property
     def phase(self):
         """Signal phase in radians."""
-        return np.angle(self.voltage_i + 1.0j * self.voltage_q)
+        return np.unwrap(np.angle(self.voltage_i + 1.0j * self.voltage_q))
```

Comment:
```suggestion
        return np.unwrap(np.arctan2(self.voltage_i, self.voltage_q))
```

I believe that is going to be used under the hood, so no need for the double conversion (we have the components, so we directly use that).

Nice that `np.unwrap()` is present in NumPy, I'd not imagined it :)